Library/Homebrew/dev-cmd/tap-new.rb

```diff
@@ -105,8 +116,20 @@ def run
 
                 - run: brew test-bot --only-tap-syntax
 
+                - name: Base64-encode GITHUB_TOKEN
```

Comment:
Should any of this be conditional on `if args.github_packages?`?
```suggestion
                - name: Base64-encode GITHUB_TOKEN for HOMEBREW_DOCKER_REGISTRY_TOKEN
```

Reply:
Good call! 
I also changed the templates from string interpolation to ERB — the readability seems better now